packages/text-area/src/TextArea/TextArea.spec.tsx

```diff
@@ -20,14 +17,33 @@ const defaultProps = {
 };
 
 function renderTextArea(props = {}) {
-  const utils = render(
-    <TextArea data-testid="text-area" label={labelProp} {...props} />,
-  );
-
-  const textArea = utils.getByTestId('text-area');
-  const label = utils.container.querySelector('label');
-  const description = utils.container.querySelector('p');
-  return { ...utils, textArea, label, description };
+  const renderUtils = render(<TextArea label={labelProp} {...props} />);
+
+  const {
+    elements: { getDescription, getErrorMessage, getInput, getLabel },
+    utils: { isDisabled, isError, inputValue },
+  } = getLGTextAreaUtils();
+
+  const textArea = getInput();
+  const label = getLabel();
+  const description = getDescription();
+
+  const rerenderTextArea = (newProps?: Partial<TextAreaProps>) => {
+    const allProps = { ...props, ...newProps };
+    renderUtils.rerender(<TextArea label={labelProp} {...allProps} />);
+  };
+
+  return {
+    ...renderUtils,
+    textArea,
+    label,
+    description,
+    getErrorMessage,
+    isDisabled,
+    isError,
+    inputValue,
```

Comment:
nit: getInputValue() i think

Reply:
inputValue doesn't indicate that it's a function

Reply:
I can rename this to `getInputValue`, what do you think?

Reply:
yes i like tha tbetter

Reply:
done